recipes/multiprocessing_on_dill/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "multiprocessing_on_dill" %}
+{% set version = "3.5.0a4" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d6d50c300ff4bd408bb71eb78725e60231039ee9b3d0d9bb7697b9d0e15045e7" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python 
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - dill
+
+test:
+  imports:
+    - multiprocessing_on_dill
+    - multiprocessing_on_dill.dummy
+
+about:
+  home: https://github.com/sixty-north/multiprocessing_on_dill
+  license: PSFL 
+  license_family: OTHER
+  summary: A friendly fork of multiprocessing which uses dill instead of pickle
+  doc_url: https://github.com/sixty-north/multiprocessing_on_dill
```

Comment:
I just saw that this project there is no license file, please add one file with the license of this project and refer it here as ``license_file``